css/src/components/notification.scss

```diff
@@ -1,31 +1,34 @@
 $notification-background: $secondary-background !default;
-$notification-padding: $spacer-5 !default;
+$notification-padding-vertical: 0.5em !default;
```

Comment:
I think there can be more vertical padding, 

![image](https://github.com/user-attachments/assets/31421152-b87c-4154-a29f-6a8c40aa4985)

Reply:
Updated